test/scales/modifiedLogScaleTests.ts

```diff
@@ -2,187 +2,241 @@
-      scale = new Plottable.Scales.ModifiedLog(base);
-      assert.deepEqual(scale.domain(), [0, base], "default domain is [0, base]");
-    });
-
-    it("can be padded", () => {
-      scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => [0, base]);
-      scale.padProportion(0);
-      let unpaddedDomain = scale.domain();
-      scale.padProportion(0.1);
-      assert.operator(scale.domain()[0], "<", unpaddedDomain[0], "left side of domain has been padded");
-      assert.operator(unpaddedDomain[1], "<", scale.domain()[1], "right side of domain has been padded");
-    });
+      it("has log() behavior at values > base", () => {
+        [10, 100, 23103.4, 5].forEach((x) => {
+          assert.closeTo(scale.scale(x), Math.log(x) / Math.log(10), 0.1);
+        });
+      });
 
-    it("autoDomain() expands single value correctly", () => {
-      scale.padProportion(0);
-      let singleValue = 15;
-      scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => [singleValue, singleValue]);
-      assert.deepEqual(scale.domain(), [singleValue / base, singleValue * base],
-        "positive single-value extent was expanded to [value / base, value * base]");
-      singleValue = -15;
-      scale.autoDomain();
-      assert.deepEqual(scale.domain(), [singleValue * base, singleValue / base],
-        "negative single-value extent was expanded to [value * base, value / base]");
-      singleValue = 0;
-      scale.autoDomain();
-      assert.deepEqual(scale.domain(), [-base, base],
-        "zero single-value extent was expanded to [base, -base]");
-    });
+      it("ensures x = invert(scale(x))", () => {
+        [0, 1, base, 100, 0.001, -1, -0.3, -base, base - 0.001].forEach((x) => {
+          assert.closeTo(x, scale.invert(scale.scale(x)), epsilon);
+          assert.closeTo(x, scale.scale(scale.invert(x)), epsilon);
+        });
+      });
 
-    it("domainMin()", () => {
-      let scale = new Plottable.Scales.ModifiedLog(base);
-      scale.padProportion(0);
-      let requestedDomain = [-5, 5];
-      scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => requestedDomain);
+      it("defaults to the [0, base] domain", () => {
+        assert.deepEqual(scale.domain(), [0, base], "default domain is [0, base]");
+      });
 
-      let minBelowBottom = -10;
-      scale.domainMin(minBelowBottom);
-      assert.deepEqual(scale.domain(), [minBelowBottom, requestedDomain[1]], "lower end of domain was set by domainMin()");
+      it("can be padded", () => {
+        scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => [0, base]);
+        scale.padProportion(0);
+        let unpaddedDomain = scale.domain();
+        scale.padProportion(0.1);
+        assert.operator(scale.domain()[0], "<", unpaddedDomain[0], "left side of domain has been padded");
+        assert.operator(unpaddedDomain[1], "<", scale.domain()[1], "right side of domain has been padded");
+      });
```

Comment:
Not sure how I can test this on the quantitative Scale. That is an abstract scale. Will see when I get there and get to pass 2.